tests/test_cot_verify.py

```diff
@@ -1263,28 +1263,44 @@ async def test_get_additional_git_action_jsone_context(github_action_link):
 
 
 @pytest.mark.parametrize(
-    "extra_env,extra_repo_definition,expected_use_parent,latest_head_sha",
+    "extra_env,extra_repo_definition,expected_use_parent,latest_head_sha,tasks_for",
     (
-        pytest.param({}, {"fork": True}, True, "somerevision", id="fork with different base repo"),
-        pytest.param({}, {"fork": False}, False, "somerevision", id="no fork with different base repo"),
+        pytest.param({}, {"fork": True}, True, "somerevision", None, id="fork with different base repo"),
+        pytest.param({}, {"fork": False}, False, "somerevision", None, id="no fork with different base repo"),
         pytest.param(
             {"MOBILE_BASE_REPOSITORY": "https://github.com/JohanLorenzo/reference-browser"},
             {"fork": True},
             False,
             "somerevision",
+            None,
             id="fork with same base repo",
         ),
-        pytest.param({}, {}, True, "someotherrevision", id="upstream task from different revision"),
+        pytest.param({}, {}, True, "someotherrevision", None, id="upstream task from different revision"),
+        pytest.param({}, {"fork": True}, True, "somerevision", "github-pull-request-untrusted", id="fork with different base repo and untrusted PR"),
+        pytest.param({}, {"fork": False}, False, "somerevision", "github-pull-request-untrusted", id="no fork with different base repo and untrusted PR"),
+        pytest.param(
+            {"MOBILE_BASE_REPOSITORY": "https://github.com/JohanLorenzo/reference-browser"},
+            {"fork": True},
+            False,
+            "somerevision",
+            "github-pull-request-untrusted",
+            id="fork with same base repo and untrusted PR",
+        ),
+        pytest.param({}, {}, True, "someotherrevision", "github-pull-request-untrusted", id="upstream task from different revision and untrusted PR"),
     ),
 )
 @pytest.mark.asyncio
 async def test_populate_jsone_context_github_pull_request(
-    mocker, mobile_chain_pull_request, mobile_github_pull_request_link, extra_env, extra_repo_definition, expected_use_parent, latest_head_sha
+    mocker, mobile_chain_pull_request, mobile_github_pull_request_link, extra_env, extra_repo_definition, expected_use_parent, latest_head_sha, tasks_for
 ):
     github_repo_mock = MagicMock()
     repo_definition = {"fork": True, "parent": {"name": "reference-browser", "owner": {"login": "mozilla-mobile"}}}
     repo_definition.update(extra_repo_definition)
     github_repo_mock.definition = repo_definition
+    if tasks_for is None:
+        tasks_for = "github-pull-request"
```

Comment:
nit: is there a reason not to just specify this in the parametrized tests?

Reply:
not really, no; updated.